larpandoracontent/LArHelpers/LArObjectHelper.h

```diff
@@ -0,0 +1,52 @@
+/**
+ *  @file   larpandoracontent/LArHelpers/LArObjectHelper.h
+ *
+ *  @brief  Header file for the object helper class.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_OBJECT_HELPER_H
+#define LAR_OBJECT_HELPER_H 1
+
+#include "Objects/Cluster.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  LArObjectHelper class
+ */
+class LArObjectHelper
+{
+public:
+    /**
+     *  @brief  TypeAdaptor
+     */
+    class TypeAdaptor
+    {
+    public:
+        /**
+         *  @brief  Get the associated position
+         *
+         *  @param  t the input object
+         *
+         *  @return the associated position
+         */
+        template<typename T>
+        static const pandora::CartesianVector GetPosition(const T &t);
```

Comment:
No need for const by value return type (as it was previously!)

Reply:
Have made this correction.  To further help me understand this, am I right in assuming a const pointer return type (ala GetCaloHit in same class) is something that makes sense as the pointer being returned points to the same thing in both the TypeAdaptor class and where this function is called and that object is const?